### ISSUE TITLE
UX: improvements of rendering on the node page

### ISSUE THREAD

**Issue by fmigneault** [DACCS-Climate/marble-website#151]
When visiting the node page (eg: https://marbleclimate.com/node.html?node=Hirondelle), a massive portion of the screen is taken to display very few details, as shown below.

![image](https://github.com/DACCS-Climate/marble-website/assets/19194484/8f0e8cdb-cdc6-42e0-9480-c330cef6510b)

Also, a large portion of the page for the node status is an empty area. 

![image](https://github.com/DACCS-Climate/marble-website/assets/19194484/9960223d-4c75-47f1-b666-8fd88bba3a9e)

When a user is looking for node details (aka the services available on that node), they have to scroll much lower to reach the relevant information. This makes for a poor user experience, as the critical information should be shown as early as possible to facilitate identification of the node capabilities. 

I think many parts of the interface should be reduced or compacted further to make the information quickly accessible. 

Above images are obtained using Firefox browser in case this is relevant for specific style rules.

**Comment by mishaschwartz** [comment]
@fmigneault 

I know that this is unrelated to the issue but there really should be images in those screenshots. Can you let me know if there are any javascript errors that you're seeing when the page loads? Also, what browser are you using?

**Comment by ShrutiK100** [comment]
@fmigneault Thank you for your feedback. We are actively working to make critical information more accessible.

**Comment by alexandercyu** [comment]
> @fmigneault
> 
> I know that this is unrelated to the issue but there really should be images in those screenshots. Can you let me know if there are any javascript errors that you're seeing when the page loads? Also, what browser are you using?

Just want to note that his screenshot looks like he's applied a dark theme to the website.  Perhaps he has an extension running that does this automatically?

**Comment by fmigneault** [comment]
@mishaschwartz 
Yes. Getting the following with Firefox 126.0.1
![image](https://github.com/DACCS-Climate/marble-website/assets/19194484/96b672e3-22af-4929-a6a2-a04cfeacea7e)

Something weird I noticed is that I can see the globe when going on the "PAVICS" node, but not on the others.

@alexandercyu 
You are correct. I'm using DarkReader. 
I've opened the site on a private window where the extension is disabled.
The JavaScript error is gone, but the rendering still looks "not quite right". I think the browser's dark theme is also interfering. 

![image](https://github.com/DACCS-Climate/marble-website/assets/19194484/edbde284-8f07-4a0c-9fc7-6edcda92a69f)
